### PR TITLE
I 105 add swagger UI to test locally

### PR DESCRIPTION
**Pull request by mahinuzzaman**
Feat: #105 
- Added swagger ui to load openapi files and update the apigateway id
- Added BearerAuth in openapi to set authorization

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?

Provide a brief overview of the changes introduced in this pull request.

## Related Issues/PRs (optional)

Link to the relevant issue(s) or provide a clear description of the problem or feature being addressed.

## How to test

- [ ] Needs regression testing (add QA needed label)

## Checklist

- [x] I have tested these changes locally and they function as expected.
- [ ] I have added or updated necessary documentation to reflect the changes.
- [ ] I have followed the project's coding style and guidelines.
- [ ] I have added appropriate test coverage to ensure the changes are functioning correctly.
- [ ] I have checked for any potential conflicts with other branches or pull requests.
- [ ] I have updated necessary OpenAPI specs

## Reviewer Request (optional)

If you have any specific requests or suggestions for reviewers, mention them here.

## Screenshots (optional)

If applicable, provide any additional screenshots or GIFs that showcase the changes made in this pull request.

## Demo Link (optional)

If applicable, provide a link to a live demo or deployment where the changes can be reviewed.

## Known Issues (optional)

If there are any known issues or limitations with the changes made in this pull request, describe them here.

## Additional Notes (optional)

Add any additional notes or comments you think are relevant to this pull request.

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -106,3 +106,10 @@ run-command-%:
 start-dev: build-runner-image localstack-start run-command-install-node-packages run-dev
 
 run-dev: run-command-build-node-all run-command-package-all run-command-tf-init run-command-tf-plan-apply run-command-tf-apply
+
+swagger-ui:
+	docker-compose -f openapi/docker-compose.yml up -d --build
```
Comment (melon-ruet) [inline Makefile:111]: docker compose 
not docker-compose

File: Makefile [hunk 2]
```diff
@@ -106,3 +106,10 @@ run-command-%:
...
+
+swagger-ui-restart:
+	docker-compose -f openapi/docker-compose.yml down
```
Comment (melon-ruet) [inline Makefile:114]: docker compose 
not docker-compose

**Review by melon-ruet — DISMISSED** [review]

File: Makefile (hunk 1 above)
Comment (mahinuzzaman) [reply Makefile]: Update in 528399b7f9aab264179391d6f5bdaedcb2c20c53

File: Makefile (hunk 2 above)
Comment (mahinuzzaman) [reply Makefile]: Updated in 528399b7f9aab264179391d6f5bdaedcb2c20c53

**Review by melon-ruet — APPROVED** [review]

**Pull request merged**